libs/core/src/components/sage-button/stories/sage-button.stories.mdx

```diff
@@ -0,0 +1,140 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+import { html, render } from 'lit-html';
+
+import { extractArgTypes } from '@pxtrn/storybook-addon-docs-stencil';
+
+<Meta title="Example/Atoms/sage-button" component="sage-button" argTypes={extractArgTypes('sage-button')} />
+
+export const Template = (args) =>
+  html` <sage-button value=${args.value} variant=${args.variant} name=${args.name} icon=${args.icon} disabled=${args.disabled} type=${args.type}> ${args.slot} </sage-button> `;
+
+# sage-button
+
+A button component to generate all versions of `sage-button` as outlined in [Figma](https://www.figma.com/file/sMbtLUHSt2vfKgKjyQ3052/%F0%9F%A7%A9-Sage-components?node-id=3%3A15&t=dIcvX1VtOu0FL0VE-0).
+
+## Usage Guidelines
+
+<table>
+  <tr>
+    <td>
+      <h3>When to use a Button</h3>
+    </td>
+    <td>
+      <h3>When to use a Link</h3>
+    </td>
+  </tr>
+  <tr>
+    <td>
+      To trigger on-page interactivity <br /> To submit a form
+    </td>
+    <td>To send a user to another URL or anchor</td>
+  </tr>
+</table>
```

Comment:
I would like to see more detail here. Remember we discussed err'ing more on the verbose side.